### PR TITLE
ENH: Add image_kwargs to report.add_epochs

### PR DESCRIPTION
**Pull request by SophieHerbst**
#### Reference issue
Needed to address 
https://github.com/mne-tools/mne-bids-pipeline/issues/848

#### What does this implement/fix?
Allow to pass image parameters when plotting epochs_images in report.add_epochs: vmin and vmax to control the color scale across participants. 

@hoechenberger can you have a look?

### CODE REVIEW

**Comment by hoechenberger** [comment]
@SophieHerbst Almost there, I believe … But listing fails:

```python
mne/report/report.py:1126:89: E501 Line too long (112 > 88)
Found 1 error.
```

**Review by hoechenberger — APPROVED** [review]
LGTM

@larsoner There's no tests so far, maybe we should add one…

File: mne/report/report.py [hunk 1]
```diff
@@ -1120,6 +1121,10 @@ def add_epochs(
             If ``True``, add PSD plots based on all ``epochs``. If ``False``,
             do not add PSD plots.
         %(projs_report)s
+        %(image_kwargs)s
```
Comment (hoechenberger) [inline mne/report/report.py:1124]:
```suggestion
```

File: mne/report/report.py [hunk 2]
```diff
@@ -1120,6 +1121,9 @@ def add_epochs(
             If ``True``, add PSD plots based on all ``epochs``. If ``False``,
             do not add PSD plots.
         %(projs_report)s
+            image_kwargs : dict | None
```
Comment (hoechenberger) [inline mne/report/report.py:1124]:
```suggestion
        image_kwargs : dict | None
```

**Comment by SophieHerbst** [comment]
@hoechenberger I don't understand what I have to do about the failed tests.

**Comment by hoechenberger** [comment]
Ah. We need to set image_kwargs to an empty dictionary if it's None. Otherwise the **image_kwargs part doesn't work

**Comment by hoechenberger** [comment]
@SophieHerbst What I mean is doing something like:

```python
if image_kwargs is None:
    image_kwargs = dict()
```

:)

**Comment by hoechenberger** [comment]
@larsoner Could you take over here please? I'll be busy until the end of the week

**Review by larsoner — APPROVED** [review]
Added a tiny test, marking for merge-when-green, thanks @SophieHerbst !

**Comment by larsoner** [comment]
Last CircleCI run worked, no idea why this one didn't but I think we can merge!

**Pull request merged**